.github/workflows/presubmit.yml

```diff
@@ -10,14 +10,14 @@ jobs:
       - uses: actions/checkout@v2
```

Comment:
These changes should be in a separate pull request.

Reply:
https://github.com/SymbiFlow/fasm/pull/33